frame/system/src/lib.rs

```diff
@@ -209,6 +209,15 @@ pub mod pallet {
 		/// except Root.
 		type BaseCallFilter: Contains<Self::RuntimeCall>;
 
+		/// The Block type used by the runtime.
```

Comment:
As per usual, I must complain about the docs :D 

You should explain what is the purpose of these types in this file. An average reader will see them, and see that they are not used in this file. This should be addressed.